src/main/java/uk/gov/hmcts/reform/prl/services/tab/alltabs/AllTabServiceImpl.java

```diff
@@ -80,10 +79,48 @@ public void updateAllTabsIncludingConfTab(CaseData caseData) {
             combinedFieldsMap.put("courtId", caseData.getCourtId());
         }
 
+        getDocumentsMap(caseData,combinedFieldsMap);
+        log.info("*******Combined fields map in tabs service********* : {}",combinedFieldsMap);
+        log.info("*******Case data Tabs service********* : {}",caseData);
+
         // Calling event to refresh the page.
         refreshCcdUsingEvent(caseData, combinedFieldsMap);
     }
 
+    private Map<String, Object> getDocumentsMap(CaseData caseData, Map<String, Object> documentMap) {
+        if (caseData.getC1ADocument() != null) {
```

Comment:
can we not just keep it like: "documentMap.put("c1ADocument", caseData.getC1ADocument());"?
Its same thing even if we doing null check, in else part we are again adding null value only